src/sonic-config-engine/minigraph.py

```diff
@@ -530,13 +530,26 @@ def parse_xml(filename, platform=None, port_config_file=None):
         if port.get('speed') == '100000':
             port['fec'] = 'rs'
 
+    # set port description if parsed from deviceinfo
     for port_name in port_descriptions:
         # ignore port not in port_config.ini
         if not ports.has_key(port_name):
             continue
 
         ports.setdefault(port_name, {})['description'] = port_descriptions[port_name]
 
+    # for the ports w/o description try to set one based on neighbor info, or just port name by default
+    for port_name, port in ports.items():
+        if not port.get('description'):
+            try:
+                neigh_host = neighbors[port_name]['name']
+                neigh_port = neighbors[port_name]['port']
+                port['description'] = neigh_host + ':' + neigh_port
```

Comment:
I don't agree with your change here. Port description shouldn't be coming from port_config.ini and certainly shouldn't default to neigh_host + neigh_port.

Reply:
This is not encouraging to specify port description via port_config.ini.

 As for default value I'd be glad to hear some suggestions.
My concern is not matching the descr advertised by lldp and snmp. The neighbor info and port name is the defaults that lldp is using. 

Let's start a discussion

Reply:
Hi @nikos-github , can you comment/approve this?